### ISSUE TITLE
Parse failure preventing other addons from updating

### ISSUE THREAD

**Issue by LucasMolander** [AcidWeb/CurseBreaker#83]
When I run `update` or `status`, it fails with this runtime error:

```
RuntimeError: Failed to parse addon data: method-dungeon-tools
The project could be removed from CurseForge or renamed. Uninstall it (and reinstall if it still
exists) to fix this issue.
```

yet that addon is perfectly proper and working.

Is there a way to add a blacklist or have CurseBreaker simply ignore "failed" addons and continue, rather than preventing it from doing anything?

**Comment by AcidWeb** [comment]
Just fix it: #82

**Comment by AcidWeb** [comment]
Apparently name change of project went bad and it crashing my bulk version check - that why it is not trying to update other addons too.
